hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/processor/SinkProcessors.java

```diff
@@ -312,13 +308,10 @@ public static ProcessorMetaSupplier writeRemoteListP(@Nonnull String listName, @
     @Nonnull
     public static <T> ProcessorMetaSupplier writeJmsTopicP(
             @Nonnull SupplierEx<? extends Connection> newConnectionFn,
-            @Nonnull FunctionEx<? super Connection, ? extends Session> newSessionFn,
             @Nonnull BiFunctionEx<? super Session, ? super T, ? extends Message> messageFn,
```

Comment:
should we still expose `Session` here?

Reply:
Yes, you need it to create the message, e.g. `session.createTextMessage("foo")`. 
The change is that Jet creates the session with parameters as needed, but user still uses it.